File: app/src/main/java/com/koala/infinitum/android_project/mainFragments/list/Factory.java
package com.koala.infinitum.android_project.mainFragments.list;

import java.util.ArrayList;

public class Factory {
    private ArrayList<String> Main1 = new ArrayList<>(), Main2 = new ArrayList<>();

    ArrayList<String> getMain1() {
        Main1.add("asasd11");
        Main1.add("asasd22");
        Main1.add("asasd33");
        Main1.add("asasd44");
        return Main1;
    }

    ArrayList<String> getMain2() {
        return Main2;
    }
}

File: app/src/main/java/com/koala/infinitum/android_project/mainFragments/list/Factory.kt
package com.koala.infinitum.android_project.mainFragments.list

import android.content.Context

class Factory(private val context: Context) {
    val Main1: ArrayList<String>
        get() {
            val result = ArrayList<String>()
            result.add("asasd1")
            result.add("asasd2")
            result.add("asasd3")
            result.add("asasd4")
            return result
        }

    val Main2: Any? = null
}